airbyte-integrations/connectors/source-notion/source_notion/streams.py

```diff
@@ -61,8 +61,16 @@ def availability_strategy(self) -> HttpAvailabilityStrategy:
         return NotionAvailabilityStrategy()
 
     @property
-    def retry_factor(self) -> float:
-        return 8
+    def retry_factor(self) -> int:
+        return 5
+
+    @property
+    def max_retries(self) -> int:
+        return 10
+
+    @property
+    def max_time(self) -> int:
+        return 80
```

Comment:
80 seconds will be used as a maximum _**total**_ amount for all backoff requests.
**_total_** means that counter starts at the moment we entry the backoff method.
Considering the retry factor is 5 and (from backoff docstring):

>  Factor to multiply the exponentiation by.

Backoff timeout will occur before even half of the planned requests(mentioned in `max_retries`) are fulfilled.

Reply:
Thank you for the clarification! I've adjusted the max_retries to 7 and max_time to 11 minutes (635 seconds backoff time + extra room in case of processing overhead) so all retry requests should be fulfilled